components/Wallet/PlaceOrder/Form/AvailableBalance.jsx

```diff
@@ -65,12 +66,13 @@ const IconButton = styled.button`
 
 export const AvailableBalance = ({ wallet, asset }) => {
   const { t } = useTranslation('place-order')
+  const price_info = asset?.price_info
   const assetBalance = useMemo(() => {
```

Comment:
Can you rename this from assetBalance to something else?

Reply:
Changing to `assetValue`